ios/Classes/FSQIPCardEntry.m

```diff
@@ -305,6 +251,52 @@ - (void)setTheme:(FlutterResult)result theme:(NSDictionary *)theme
     result(nil);
 }
 
+- (void)startBuyerVerificationFlow:(FlutterResult)result buyerActionString:(NSString *)buyerActionString moneyMap:(NSDictionary *)moneyMap locationId:(NSString *)locationId contactMap:(NSDictionary *)contactMap paymentSourceId:(NSString *)paymentSourceId
```

Comment:
Similar to the comment on Android side, can we use a separate Module?

Reply:
I like the helper methods you created, we may create a util for different modules.

Reply:
> Similar to the comment on Android side, can we use a separate Module?

Sure.

Reply:
> I like the helper methods you created, we may create a util for different modules.

Thanks :)
Yeah, we can re-use same code wherever required.